src/core/constants.js

```diff
@@ -280,8 +275,6 @@ export const MOZILLA_COLLECTIONS_EDIT = 'Admin:Curation';
 export const FEATURED_THEMES_COLLECTION_EDIT = 'Collections:Contribute';
 // The slug for the special Featured Themes collection.
 export const FEATURED_THEMES_COLLECTION_SLUG = 'featured-personas';
-// Can confirm approval of automatically approved add-ons.
-export const ADDONS_POST_REVIEW = 'Addons:PostReview';
```

Comment:
I still see this perm mentioned in addons-server.

Reply:
`Addons:Review` and `Addons:PostReview` were merged in https://github.com/mozilla/addons-server/issues/14782.

Reply:
https://github.com/mozilla/addons-server/blob/4e2c1f7c9c15fba477be9e6aefbca5b940097a74/src/olympia/reviewers/utils.py#L433

Reply:
Hm, looks like some comments and variable names where not updated. Eventually, this actually points to `permission_post_review` which can have various values, but never has `Addons:PostReview`, since that constant was removed. /cc @diox